engine/installation/linux/linux-postinstall.md

```diff
@@ -97,6 +97,69 @@ For information about the different storage engines, see
 The default storage engine and the list of supported storage engines depend on
 your host's Linux distribution and available kernel drivers.
 
+## Enable experimental features
+
+Some features of Docker are introduced as experimental features for one or more
+releases. Previously, you needed to download and install an
+**experimental build** to use these features. In Docker 1.13 and higher,
+experimental features are included in all stable Docker builds and packages,
+but disabled by default. To enable experimental features, start the Docker
+daemon (`dockerd`) with the `--experimental` flag. You can do this temporarily
+or permanently.
+
+> **Note**: Docker for Windows and Docker for Mac ship with experimental
+> features enabled by default.
```

Comment:
I think that's only for the beta releases, stable releases have them disabled? But you can enable them in the UI

@justincormack ?